packages/bot-web-ui/src/components/bot-notification-messages/bot-notification-messages.scss

```diff
@@ -10,7 +10,7 @@
     }
     &--is-panel-open {
         .notification-messages {
-            transform: translate3d(calc(-350px), 0, 0);
+            transform: translate3d(calc(-368px), 0, 0);
```

Comment:
setting position of the notifications according to the position of the Info Panel and Run Panel